### PR TITLE
feat: implement rainfall and vegetation maps

### PR DESCRIPTION
**Pull request by jschoedl**
Due to the CORS of the WFP APIs, the map doesn't work in the demo. During development I used a Plug-In that temporarily disabled the CORS guidelines.
Bildschirmfoto vom 2024-11-18 20-56-33
Bildschirmfoto vom 2024-11-18 20-56-47
Bildschirmfoto vom 2024-11-18 20-57-04
Bildschirmfoto vom 2024-11-18 20-56-56

### CODE REVIEW

**Review by marinovl7 — APPROVED** [review]
LGTM for me! @Lukas0912 will do the final approval and merge!

File: src/operations/map/MapOperations.ts [hunk 1]
```diff
@@ -57,6 +58,16 @@ export class MapOperations {
               'fill-opacity': ['case', ['boolean', ['feature-state', 'hover'], false], 0.7, 1],
```
Comment (Lukas0912) [inline src/operations/map/MapOperations.ts:58]: I think this line doesn't have any effect anymore now

**Review by Lukas0912 — CHANGES_REQUESTED** [review]
Very nice improvement of the layer system from before, I would just suggest to add another layer on top with the deactivated non interactive countries and change the base layer to a uniformly colored base layer. Right now the differentiation of selectable and non selectable countries is not possible anymore since the overlay isn't transparent.

**Comment by jschoedl** [comment]
Bildschirmfoto vom 2024-11-19 10-26-40
Bildschirmfoto vom 2024-11-19 10-26-48

**Pull request merged**